### DISCUSSION TITLE
Are these functions also available in BamuStudio?

### DISCUSSION THREAD

**Discussion by 3D-Frank** [OrcaSlicer/OrcaSlicer#178] [General]
Hi!
Can I also activate these two important functions from the SuperSlicer in BambuStudio?
Kind regards!
<img width="380" alt="Bildschirm­foto 2023-01-14 um 18 35 37" src="https://user-images.githubusercontent.com/97818762/212488189-d2f040dd-3113-484c-88b8-71f71c1b2793.png">

**Comment by hliebscher** [comment]
both working in BambuStudio SoftFever Version

**Reply by 3D-Frank** [reply]
👍Thanks!
